Application/EdFi.Ods.Api.Common/EdFi.Ods.Api.Common.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;net48</TargetFrameworks>
+    <AssemblyName>EdFi.Ods.Api.Common</AssemblyName>
+    <RootNamespace>EdFi.Ods.Api.Common</RootNamespace>
+    <Copyright>Copyright © 2019 Ed-Fi Alliance, LLC and Contributors</Copyright>
+    <TreatErrorsAsWarning>true</TreatErrorsAsWarning>
+    <RestorePackages>true</RestorePackages>
+    <OutputType>Library</OutputType>
+    <AssemblyVersion>0.0.0</AssemblyVersion>
```

Comment:
We don't usually include versions in these csprojs, the build/pack steps take care of that.